x-pack/test/usage_collection/config.ts

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
I think that this new folder of functional tests can be used as well when implementing #83704